read_unix.go

```diff
@@ -9,6 +9,9 @@ import (
 	"golang.org/x/crypto/ssh/terminal"
 )
 
+// LINE_SEP is the separator for windows or unix systems
+const LINE_SEP = "\n"
```

Comment:
Thanks, but can you change var name to `LineSep` ? ref. https://golang.org/doc/effective_go.html#mixed-caps